manifests/overlays/apps/base/aikit/oneapi-aikit-notebook-quickstart.yaml

```
@@ -76,44 +76,44 @@ spec:
-        3. Choose the `Intel SKLearn, XGBoost, & Modin`  JupyterHub Kernel Environment to launch the notebook in.
+        3. Choose the `Intel SKLearn, XGBoost, & Modin`  Jupyter Kernel Environment to launch the notebook in.
       review:
         instructions: |-
           #### Verify that the notebook launched successfully
-          Do you have a notebook tab for the `IntelPython_XGBoost_daal4pyPrediction.ipynb` notebook in your browser in the `Intel SKLearn, XGBoost, & Modin`  JupyterHub Kernel Environment?
+          Do you have a notebook tab for the `IntelPython_XGBoost_daal4pyPrediction.ipynb` notebook in your browser in the `Intel SKLearn, XGBoost, & Modin`  Jupyter Kernel Environment?
         failedTaskHelp:
           This task is not verified yet. Try the task again.
       summary:
-        success: You loaded the example notebook in the correct AI Kit JupyterHub Kernel environment.
+        success: You loaded the example notebook in the correct AI Kit Jupyter Kernel environment.
         failed: Try the steps again.
     - title: Run an Intel® oneAPI AI Analytics Toolkit sample notebook
       description: |-
         ### Running an Intel® oneAPI AI Analytics Toolkit sample
         1. Run the `IntelPython_XGBoost_daal4pyPrediction.ipynb` notebook cell by cell, taking care not to skip any steps. Once you have completed the entire notebook, you will have learned about some AI Kit optimizations and fundamentals.
-        2. (Optional) You can proceed with this step and the previous step with any of the either samples in the correct AI Kit JupyterHub Kernel environment to learn more about other AI Kit optimizations. Use the `Intel SKLearn, XGBoost, & Modin` JupyterHub Kernel Environment for the `IntelPython_XGBoost_daal4pyPrediction.ipynb` notebook example, and the `Intel TensorFlow & Quantization` JupyterHub Kernel Environment for the `lpot_sample_tensorflow.ipynb` notebook example.
+        2. (Optional) You can proceed with this step and the previous step with any of the either samples in the correct AI Kit Jupyter Kernel environment to learn more about other AI Kit optimizations. Use the `Intel SKLearn, XGBoost, & Modin` Jupyter Kernel Environment for the `IntelPython_XGBoost_daal4pyPrediction.ipynb` notebook example, and the `Intel TensorFlow & Quantization` Jupyter Kernel Environment for the `lpot_sample_tensorflow.ipynb` notebook example.
       review:
         instructions: |-
           #### Verify that the sample notebook ran successfully
           Did all cells of the notebook run successfully and printed out the statement "[CODE\_SAMPLE\_COMPLETED\_SUCCESSFULLY]" at the end?
         failedTaskHelp:
           This task is not verified yet. Try the task again.
       summary:
-        success: You have run the example notebook in the correct AI Kit JupyterHub Kernel environment.
+        success: You have run the example notebook in the correct AI Kit Jupyter Kernel environment.
```

```suggestion
        success: You have run the example notebook in the correct AI Kit Jupyter Kernel Environment.
```

Initcap for Jupyter Kernel Environment (not environment)? If so, search and replace globally.